lib/Common/ConfigFlagsList.h

```diff
@@ -344,6 +344,7 @@ PHASE(All)
         PHASE(StackFramesEvent)
 #endif
         PHASE(PerfHint)
+        PHASE(TypeShare)
```

Comment:
Can you give this a less generic name? It doesn't control type-sharing, only a very specific instance of type-sharing.